domain/service/servicedetails.go

```diff
@@ -29,6 +29,7 @@ type ServiceDetails struct {
 	Description     string
 	PoolID          string
 	ParentServiceID string
+	Parent          *ServiceDetails `json:",omitempty"`
```

Comment:
I don't like that, because that implies we have to make 2 calls to get the data we need.  The purpose of this is so the function only needs to make one call.